### PR TITLE
Add new Chinese translation

### PR DESCRIPTION
**Pull request by felixqu**
Add new Chinese translation

### CODE REVIEW

File: src/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -443,4 +443,168 @@
...
+  "addedSite": {
+    "message": "已添加的网站",
+    "description": "Added site"
+  },
+  "editedSite": {
+    "message": "已编辑的网站",
+    "description": "Edited site"
+  },
+  "deleteSiteConfirmation": {
+    "message": "Are you sure you want to delete this site?",
```
Comment (kspearrin) [inline src/_locales/zh_CN/messages.json]: Translation here?

File: src/_locales/zh_CN/messages.json [hunk 2]
```diff
@@ -443,4 +443,168 @@
...
+  "verificationCodeRequired": {
+    "message": "必须填写验证码。",
+    "description": "Verification code is required."
+  },
+  "valueCopied": {
+    "message": " 已复制",
+    "description": "' copied'. This is part of a sentence to be sure to leave the space prefix. For example: 'Password copied'"
+  },
+  "autofillError": {
+    "message": "无法在此页面自动填充所选择的网站。请手工 复制/粘贴 您的用户名、密码",
```
Comment (kspearrin) [inline src/_locales/zh_CN/messages.json]: This seems incorrect?

**Comment by felixqu** [comment]
I've fixed and re-pull

File: src/_locales/zh_CN/messages.json [hunk 3]
```diff
@@ -443,4 +443,168 @@
     "message": "安全",
     "description": "Security"
   }
```
Comment (kspearrin) [inline src/_locales/zh_CN/messages.json]: need a comma here now

**Comment by kspearrin** [comment]
Thanks again!

**Comment by felixqu** [comment]
My pleasure.

**Pull request merged**